apps/pie-storybook/stories/pie-checkbox-group.stories.ts

```diff
@@ -1,36 +1,91 @@
 import { html } from 'lit';
-
+import { ifDefined } from 'lit/directives/if-defined.js';
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-checkbox-group';
-import { CheckboxGroupProps } from '@justeattakeaway/pie-checkbox-group';
+import { CheckboxGroupProps, defaultProps, statusTypes } from '@justeattakeaway/pie-checkbox-group';
 /* eslint-enable import/no-duplicates */
 
 import { type StoryMeta } from '../types';
 import { createStory } from '../utilities';
 
+import '@justeattakeaway/pie-link';
+import '@justeattakeaway/pie-checkbox';
+
 type CheckboxGroupStoryMeta = StoryMeta<CheckboxGroupProps>;
 
-const defaultArgs: CheckboxGroupProps = {};
+const defaultArgs: CheckboxGroupProps = {
+    ...defaultProps,
+};
 
 const checkboxGroupStoryMeta: CheckboxGroupStoryMeta = {
     title: 'Checkbox Group',
     component: 'pie-checkbox-group',
-    argTypes: {},
+    argTypes: {
+        label: {
+            description: 'The visible label for the checkbox group',
+            control: 'text',
+        },
+        status: {
+            description: 'The status of the checkbox group component / assistive text. Can be default, success or error.',
+            control: 'select',
+            options: statusTypes,
+            defaultValue: {
+                summary: defaultProps.status,
+            },
+        },
+        disabled: {
+            description: 'If true, disables the whole checkbox group. If you need to disable only one checkbox pass the disabled prop to the needed checkbox.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.disabled,
+            },
+        },
+        assistiveText: {
+            description: 'An optional assistive text to display below the checkbox group.',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+    },
     args: defaultArgs,
     parameters: {
         design: {
             type: 'figma',
-            url: '',
+            url: 'https://www.figma.com/design/aD4m0j97Ruw8Q4S5lED2Bl/Checkbox-audit?node-id=9938-14138&t=1V1VEYmtArFih6AX-4',
         },
     },
 };
 
 export default checkboxGroupStoryMeta;
 
-// TODO: remove the eslint-disable rule when props are added
-// eslint-disable-next-line no-empty-pattern
-const Template = ({}: CheckboxGroupProps) => html`
-    <pie-checkbox-group></pie-checkbox-group>
+const Template = ({
+    label,
+    assistiveText,
+    status,
+    disabled,
+}: CheckboxGroupProps) => html`
+    <p>Please note, the checkboxes are separate components. See
+    <pie-link href="/?path=/story/checkbox--default">pie-checkbox</pie-link>.</p>
+    <pie-checkbox-group
+        label="${ifDefined(label)}"
+        assistiveText="${ifDefined(assistiveText)}"
+        status=${ifDefined(status)}
+        ?disabled="${disabled}"
+    >
+        <pie-checkbox
```

Comment:
Not specifically to do with this file, but I noticed when moving through focusing checkboxes, the focus can get a bit hidden by the next checkbox. Not something necessarily to fix in this PR, but I'd expect the focus to be above the checkboxes.

![Screenshot 2024-07-12 at 14 10 39](https://github.com/user-attachments/assets/e5f241b6-7794-4f58-adad-673c75de986a)

Reply:
The styling for the component will come in a separate PR and there will be enough space between checkboxes 👍